contentful_management/editor_interfaces_proxy.py

```diff
@@ -30,14 +30,21 @@ def _resource_class(self):
 
     def all(self):
         """
-        Gets the default Editor Interface.
+        Gets the default editor interface.
         """
 
         return super(EditorInterfacesProxy, self).all()
 
     def find(self):
         """
-        Gets the default Editor Interface.
+        Gets the default editor interface.
+        """
+
+        return self.all()
+
+    def default(self):
+        """
+        Gets the default editor interface.
```

Comment:
I'm probably missing some context with editor interfaces, but`def default` and _Gets the default editor interface_ imply a single result, but I guess `return self.all()` returns a collection, am I correct?